HW2_Zolotenkova/calculator.py

```diff
@@ -0,0 +1,24 @@
+def summa(a, b):
+    answer = float(a) + float(b)
+    return (answer)

```

Comment:
Скобки у return не нужны

```suggestion
    return answer

```